packages/cmf/src/api.js

```diff
@@ -38,5 +44,7 @@ export default {
 	expression,
 	route,
 	registry,
+	registerInternals,
+	sagas,
```

Comment:
sagas & saga feel me strange, could be ambiguous
could one of those ( sagas ) be renamed or moved maybe ?

Reply:
merged into one